packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -419,10 +412,22 @@ export const getLinksSupplier = (
       },
     };
 
-    info.href = replaceVariables(info.href, variables);
-    info.title = replaceVariables(info.title, variables);
-    info.href = locationUtil.processUrl(info.href);
+    if (link.internal) {
+      // For internal links at the moment only destination is Explore.
+      return mapInternalLinkToExplore(link, variables, {} as any, field);
```

Comment:
To support the internal links in dahboard table panel I needed to add it here. Find the general data flow of links confusing so personally don't think it should be done here as it isn't tied to overrides but not sure what would be a better place right now.

Reply:
I think this could be added to ValueLinkConfig , so callers of getLinks can provide a custom handler for how internal links should be processed / evaluated

Reply:
The problem is how to get it there. This is called by the visualisation but I do not want the visualisation to know if it is inside the explore or dashboard or how to deal with the link. I think that should responsibility of the dashboard code or explore code. So I would have to pass it from there somehow anyway.